.idea/misc.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file adding should be reverted, add `/.idea` to your .git/info/exclude using.

`echo "/.idea" >> .git/info/exclude` in your command shell.